Client/Frontend/Browser/TabManager.swift

```diff
@@ -39,6 +39,29 @@ class TabManager {
         return tabs[index]
     }
 
+    func selectTab(webView: WKWebView) {
```

Comment:
To keep this a little simpler and to avoid duplicating tab selection handling, please change this to <code>getTab(forWebView: WKWebView) -> Browser</code>, then call <code>selectTab(tab: Browser?)</code> on the result.